pkg/ecdsa/tss/protocol_ready.go

```diff
@@ -97,3 +143,22 @@ func readyProtocol(
 		return fmt.Errorf("unexpected context error: [%v]", ctx.Err())
 	}
 }
+
+func memberIDToAddress(
+	memberID MemberID,
+	groupID string,
```

Comment:
We need this property to construct error message only. We shouldn't pass it.

Reply:
That was the intention here. If we do not pass it, how to log it? Or shouldn't we log it? We can have multiple key generations and signings happening at the same time, having the context of keep address for which the operation failed could be helpful.

Reply:
We could add `groupID` when handling the error returned by this function, as we already do:
https://github.com/keep-network/keep-ecdsa/blob/509a5a431c540ac58bb0e4245109e5ed6c10521a/pkg/ecdsa/tss/protocol_ready.go#L61
https://github.com/keep-network/keep-ecdsa/blob/509a5a431c540ac58bb0e4245109e5ed6c10521a/pkg/ecdsa/tss/protocol_ready.go#L119

Reply:
That's a good idea, assuming we can express it clearly for operators who will have to read this error and act upon it. The best I could come up with is:
```
could not convert member ID to address for a member of keep [0x12345ABC]: [could not get public key for member with ID [987654321]: [something went wrong]
```
Already implemented in https://github.com/keep-network/keep-ecdsa/pull/654/commits/883121a3a1f0475068326085373fc110eec3c6db.